src/cpp/r/R/Api.R

```diff
@@ -368,6 +376,15 @@
       url = NULL)
 })
 
+.rs.addApiFunction("writePreference", function(name, value) {
```

Comment:
@jjallaire I think you already figured out the motivation of this change from the `rstudioapi` PR already: To allow `sparklyr`/users to persist their `dplyr` or `DBI` preference in the UI.